facade/service.go

```diff
@@ -1245,129 +1253,72 @@ func (f *Facade) scheduleService(ctx datastore.Context, tenantID, serviceID stri
 		return 0, err
 	}
 
-	serviceScheduler := func() (int, error) {
```

Comment:
Note: This was how we handled emergency shutdown previously - it's now performed in servicestatemanager.